docs/source/noaa_csp.rst

```diff
@@ -40,160 +51,252 @@ Login to the NOAA CSP
+Note, current global-workflow still using CentOS built spack-stack.
+It will be updated to Rocky 8 soon.
 
 .. image:: _static/noaacsp_instance_1.png
    
-Next, the mandatory attributes for the respective instance must be
-defined as shown in the illustration below.
+Next, click "Forklatest" as shown in the red-circle.
 
 .. image:: _static/noaacsp_instance_2.png
-
-The annotated attributes and their respective descriptions are as
-follows.
-
-1. A unique name for the instance. Best practices suggest one that is
-   clear, concise, and relevant to the application.
-2. A short description of the instance, i.e., ``This instance supports
-   this <task name> task.``
-3. Tag(s) describing and identifying the respective instance. These
-   allow for improved bookkeeping, especially when a user has multiple
-   or concurrent instance types.
-
-Next, the cluster is defined as shown in the following illustration.
+   
+Please provide a unique name (box pointer by the red-arrow) for the instance.
+Best practices suggest one that is clear, concise, and relevant to the application.
+Click :red-text:`Fork` (in the red-circle) to fork an instance.
 
 .. image:: _static/noaacsp_instance_3.png
 
-The NOAA Parallel Works initiative currently provides 2 CSPs for the
-global-workflow; **AWS** (Amazon Web Services) and **Azure**
-(Microsoft Azure). Existing clusters may also be modified. However
-this is neither recommended or supported.
+Now, an instance is forked, and it is time to configure the cluster.
+
+1. Select a "Recource Account", usually it is: NOAA AWS Commercial Vault.
+2. Select a "Group". Which will be something like: ca-epic, ca-sfs-emc, etc.
+3. Copy and paste you Public key (.ssh/id_rsa.pub, .ssh/id_dsa.pu from your laptop).
+4. Modify "User Bootstrap". If you are not use "ca-epic" group, please UNCOMMENT line 2.
+5. Keep "health Check" as it is.
 
-Finally, when satisfied with the CSP instance configure, click ``Add
-Resource`` as illustrated below.
+Click "Save Changes" at top-right as shown in read circle.
 
 .. image:: _static/noaacsp_instance_4.png
 
+The NOAA ParallelWorks currently provides 3 CSPs:
+**AWS** (Amazon Web Services), **Azure** (Microsoft Azure),
+and **GCP** (Google Cloud Platform).
+Existing clusters may also be modified.
+Better practice is to fork from Marketplace with something similar to your requests.
+
 ******************************
-Configure the NOAA CSP Cluster
+Add CSP Lustre Filesystem
 ******************************
 
-Navigate to the tab and locate the CSP instance configured in the
-previous section and click on the link, `globalworkflowdemo` for this
-example.
+To run global-workflow on CSPs, we need to attached lustre filesystem as run directory.
+Figure, we need to add/define our lustre filesystem, to do so,
+navigate to the middle of noaa.parallel.works web-site left panel and select "Lutre" 
+as pointed by red-arrow, and then click "AddStorage" at top-right as in the red-circle.
```

Comment:
```suggestion
To run global-workflow on CSPs, we need to attach the ``/lustre`` filesystem as run directory.
First, we need to add/define our ``/lustre`` filesystem. To do so,
navigate to the middle of the noaa.parallel.works website left side panel and select "Lustre" (see the red arrow in :numref:`Figure %s <noaacsp_lustre_1>`), and then click "Add Storage" at the top right as shown in the red-circle.
```

Reply:
In line 111, you can crosslink to the anchor I added below. You can do that for any image. You might also want to add titles to the different images? Not essential, but helpful for readers.